### ISSUE TITLE
Vulnerability description field is being stripped from newlines when creating CSV file. Make that an option!

### ISSUE THREAD

**Issue by gister9000** [infobyte/faraday#409]
## Issue Type
 - Feature Idea

Not sure if it's a bug or intentional. Likely intentional. 

## Faraday version
Release v3.11

## Steps to reproduce
1. Write/import a nice vuln description like this one:
![image](https://user-images.githubusercontent.com/55654477/92913878-6f854f00-f42b-11ea-80c4-d7488e015663.png)

2. Download CSV and see that newlines were stripped:
![image](https://user-images.githubusercontent.com/55654477/92913983-862ba600-f42b-11ea-8c63-e5f20f7b65b2.png)

## Expected results
I expected to find "\n" characters where newlines were in the description.
 
Why don't you allow users to configure whether or not they want their newlines to be stripped or not? I think that most users like to preserve the newline they entered since they entered it for a reason.

**Comment by gister9000** [comment]
Do you want me to contribute by changing this myself? BTW, why can't I edit references after creating the vulnerability? Should I change that as well?

**Comment by EricHorvat** [comment]
Hi @gister9000 !

We will see this an issue, and problaby fixed in the next release. But we're open to community PRs! So if you fix this before us, surely would be merged!

Best regards

Horvat Eric

**Comment by llazzaro** [comment]
Hi @gister9000,

```
why can't I edit references after creating the vulnerability? 
```

Yes, you need to select the vuln and then click on the `edit` button.

**Comment by gister9000** [comment]
Hi @EricHorvat,
thanks for acknowledging the issue. I will probably wait for the next release though :D 

Hi @llazzaro, 
thanks! I was clicking the vulnerability and not the edit button. That solves it!

Best regards,
Filip Jelić

**Comment by aenima-x** [comment]
@gister9000 This will be fixed in the next release (3.15)
